EI.yaml

```diff
@@ -0,0 +1,132 @@
+destination:
+  description: Earlham Institute Training Material
+  name: EI Training Material
```

Comment:
@bgruening  Wondering if I should have used "Galaxy courses" instead, to not pollute the top-level libraries?

Reply:
Yes, that would be better imho!
Can you change that?

Reply:
Sure, https://github.com/usegalaxy-eu/shared-data/pull/9